src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -302,7 +302,10 @@ public void run() {
         // one isn't there
         return this.job.scheduleBuild2(job.getQuietPeriod(), cause, new ParametersAction(values), findPreviousBuildForPullId(pullIdPv));
     }
-    
+
+    private String escapeText(String text) {
+        return text.replace("\n", "\\n").replace("\r", "\\r").replace("\"", "\\\"");
```

Comment:
Should the \ be replaced first?  Otherwise you would end up with: `\n` -> `\\n` -> `\\\\\\n`